src/_pytest/python_api.py

```diff
@@ -60,6 +91,13 @@ def __init__(self, expected, rel=None, abs=None, nan_ok: bool = False) -> None:
     def __repr__(self) -> str:
         raise NotImplementedError
 
+    def repr_compare(self, other_side, verbosity_level) -> typing.List[str]:
```

Comment:
Is it possible to type annotate this method fully?

Reply:
Yes. Will do.